code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -561,16 +561,16 @@
 	icon_state = "petcollar"
 	item_color = "petcollar"
 	var/tagname = null
-	var/obj/item/weapon/card/id/access_id
+	var/obj/item/weapon/card/id/id
```

Comment:
The reason I changed access_id to id, wasn't because I felt like it.. rather, because at the time I thought that might have had to do with id's functioning through other objects. Referencing another object that using ID's in this manner (PDA's), I noticed that the var for holding an id was /id/id. Originally I thought that it was specifically referencing id/id in the code.. but learned recently I was mistaken. I kept it afterwards for consistency reasons (PDA's using the same value), although I prefer access_id.

I am still re-learning, so apologies. And.. you have a point about the changelog and it really not fixing anything. Though I didn't keep it for arbitrary reasons. Easily changed back if needed.